packages/config/config/devices/0x0063/ge_12730_zw4002.json

```diff
@@ -15,13 +15,29 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"associations": {
+		"1": {
+			"label": "Group 1",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Local Load",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Double Tap",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
```

Comment:
This doesn't seem to be doing anything. I downloaded this file, mapped it into my docker container, and did a re-interview, and no groups show up.

docker-compose:
```
    volumes:
      - ./zjs/data:/data
      - ./zjs/settings.json:/data/settings.json:ro
      - "./zjs/custom/ge_12730_zw4002.json:/usr/src/app/node_modules/@zwave-js/config/config/devices/0x0063/ge_12730_zw4002.json"
```

Association CC is not listed as a supported CC:
```
11:06:04.207 CNTRLR « [Node 021] node info received
                      supported CCs:
                      · Multilevel Switch
                      · All Switch
                      · Configuration
                      · Version
                      · Manufacturer Specific
                      controlled CCs:
```

No groups in the UI:
![image](https://user-images.githubusercontent.com/791794/111041542-78aca400-83ed-11eb-9374-2475d8d9ad7e.png)

The configuration settings were updated correctly so I know I mapped the file correctly:
![image](https://user-images.githubusercontent.com/791794/111041558-9417af00-83ed-11eb-8539-c77ea0945f47.png)

Reply:
Did you restart to rebuild the index?

Reply:
The quotes don't matter. Please explain how the Configuration parameters would have changed if I didn't update the file.

The 14287 is not the same thing as the 12730.

Reply:
I edited my post as I missed that part of yours. They are both build on the zw4002 hardware according to the specs.

Reply:
Did you not notice you have edited both a config file for 12730 and 14287 in this PR? They have different product ids. The 12730 is Z-Wave Classic and the 14287 is Z-Wave Plus. The zw4002 identifier is absolutely irrelevant.

https://products.z-wavealliance.org/products/1202?selectedFrequencyId=2
https://products.z-wavealliance.org/products/2103?selectedFrequencyId=2

Reply:
They are different models. They share the same Jasco model base and the SAME FCC ID. the 14287 is a rebranded 12730 with an upgraded firmware adding zwave plus.

Reply:
That's great, but one is Z-Wave Classic and one is Z-Wave Plus. That makes them not the same with respect to Z-Wave, which is what matters.

Reply:
I'm not sure why everything needs to be so argumentative. Do you want to take over maintenance of the device files?

Reply:
Look, I'm done arguing about this. I'm not adding the compat flag, its already there. It isn't the subject of this PR. If you think it should be removed, push a PR doing so and then we can talk about it on your PR.

Reply:
Look, I'm done arguing about this. I'm not adding the compat flag, it already exists. If you think it should be removed push your own PR and we'll talk about it there. It isn't the subject of this PR.